### DISCUSSION TITLE
<flashwriter Ex   alternative app>

### DISCUSSION THREAD

**Discussion by vijayaprakash-web** [community/community#130337] [Programming Help]
### Select Topic Area

Product Feedback

### Body

required alternative free app of flashwriter Ex because it's paid version . I want flash boot software for SH7055 cpu

**Comment by alexisseurin** [comment]
You have OpenOCD (Open On-Chip Debugger) which supports a wide range of CPUs and microcontrollers.

**Reply by vijayaprakash-web** [reply]
Openocd support SH7055s

**Comment by Rishi-Sudhakar** [comment]
Flashmagic, ST link, or just Renesas apps?

**Reply by vijayaprakash-web** [reply]
flashmagic is not support Sh7055 cpu

**Comment by HemalathaT2** [comment]
h8write, mes.osdn.jp/h8/writer-j.html

**Reply by vijayaprakash-web** [reply]
Thanks for your comments I try to download but this website not open kindly suggest best way  for downloading H8write

**Reply by HemalathaT2** [reply]
https://netbsd.pkgs.org/10/netbsd-aarch64/kz_h8write-0.2.1.tgz.html

**Reply by vijayaprakash-web** [reply]
download the software with the above link but that file is not installed. can you guide 
![Capture](https://github.com/community/community/assets/174176842/bc008f45-2ef5-49ac-8c4e-5d0ac812959e)

 please
